tsconfig.json

```diff
@@ -60,7 +60,8 @@
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
     "pretty": true,
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
```

Comment:
This is the reason why your build location has changed. As long as all source files are in a single directory (i.e. src), tsc will leave out that directory in the dist directory (which is what we want).

You included this option to be able to require the package.json file and fetch the version from there. I would remove this option and read the package file in another way or, if impossible, duplicate the version number or leave out the version command.